src/main/java/com/google/sps/servlets/RepAnswerServlet.java

```diff
@@ -0,0 +1,43 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.sps.data.InsertAndUpdate;
+import com.google.sps.data.QueryDatastore;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.EntityNotFoundException;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Key; 
+import com.google.appengine.api.datastore.KeyFactory;
+
+@WebServlet ("/rep_answer")
+public class RepAnswerServlet extends HttpServlet{
+    private static final String POST_ENTITY_TYPE = "Post";
+    private static final String POST_QUESTION = "Question";
+    private static final String POST_ANSWER = "Answer";
+    private static final String POST_REPLIES = "Replies";
+
```

Comment:
I made a constants class so just import that. (it's called Constants.java so all of these become Constants.POST_ANSWER, etc. )

Reply:
just added Constants and adjusted everything - hopefully should minimize conflicts